static/app/views/issueList/draggableTabBar.tsx

```diff
@@ -171,7 +193,23 @@ export function DraggableTabBar({
             hidden={tab.key === 'temporary-tab' && !showTempTab}
           >
             <TabContentWrap>
-              {tab.label}
+              <EditableTabTitle
+                label={tab.label}
+                isEditing={
+                  isEditingTabTitle.editingTabKey === tab.key &&
+                  isEditingTabTitle.isEditing
+                }
+                setIsEditing={isEditing =>
+                  setIsEditingTabTitle({
+                    isEditing,
+                    editingTabKey: isEditing ? tab.key : undefined,
+                  })
+                }
+                onChange={newLabel => handleOnTabRenamed(newLabel.trim(), tab.key)}
+                inputRef={
+                  isEditingTabTitle.editingTabKey === tab.key ? tabLabelRef : undefined
+                }
```

Comment:
I'm attaching the single reference to _only_ the title that is being renamed, rather than creating a reference for each tab title from the start. This feels like it could be one of those things that has unforeseen consequences, lmk if that's the case

Reply:
Do you need the ref for anything in the future? If not I wouldn't keep track of any refs